test/utils/input/utils.js

```diff
@@ -0,0 +1,426 @@
+			return expect(result).to.eventually.eql({
+				data: null,
+				passphrase: null,
+				password: stdContents,
+				secondPassphrase: null,
+			});
+		});
+
+		it('should resolve second passphrase', () => {
+			const options = {
+				secondPassphraseIsRequired: true,
+			};
+			const result = inputUtils.getStdIn(options);
+			return expect(result).to.eventually.eql({
+				data: null,
+				passphrase: null,
+				password: null,
+				secondPassphrase: stdContents,
+			});
+		});
+
+		it('should resolve data', () => {
+			const options = {
+				dataIsRequired: true,
+			};
+			const result = inputUtils.getStdIn(options);
+			return expect(result).to.eventually.eql({
+				data: stdContents,
+				passphrase: null,
+				password: null,
+				secondPassphrase: null,
+			});
+		});
+	});
+
+	describe('#getPassphraseFromPrompt', () => {
+		const displayName = 'password';
+		beforeEach(() => {
+			return sandbox.stub(inquirer, 'prompt');
+		});
+
+		it('passphrase should equal to the result of the prompt', async () => {
+			const promptResult = { passphrase: '123' };
+			inquirer.prompt.resolves(promptResult);
+			const passphrase = await inputUtils.getPassphraseFromPrompt({
+				displayName,
+			});
+			return expect(passphrase).to.equal(promptResult.passphrase);
+		});
+
+		it('should prompt once with shouldRepeat false', async () => {
+			const promptResult = { passphrase: '123' };
+			inquirer.prompt.resolves(promptResult);
+			await inputUtils.getPassphraseFromPrompt({
+				displayName,
+			});
+			return expect(inquirer.prompt).to.be.calledWithExactly([
+				{
+					name: 'passphrase',
+					type: 'password',
+					message: `Please enter ${displayName}: `,
+				},
+			]);
+		});
+
+		it('should prompt twice with shouldRepeat true', async () => {
+			const promptResult = { passphrase: '123', passphraseRepeat: '123' };
+			inquirer.prompt.resolves(promptResult);
+			await inputUtils.getPassphraseFromPrompt({
+				shouldRepeat: true,
+				displayName,
+			});
+			return expect(inquirer.prompt).to.be.calledWithExactly([
+				{
+					name: 'passphrase',
+					type: 'password',
+					message: `Please enter ${displayName}: `,
+				},
+				{
+					name: 'passphraseRepeat',
+					type: 'password',
+					message: `Please re-enter ${displayName}: `,
+				},
+			]);
+		});
+
+		it('should reject with error when repeated passphrase does not match', () => {
+			const promptResult = { passphrase: '123', passphraseRepeat: '456' };
+			inquirer.prompt.resolves(promptResult);
+			return expect(
+				inputUtils.getPassphraseFromPrompt({ shouldRepeat: true, displayName }),
+			).to.be.rejectedWith(
+				ValidationError,
+				'Password was not successfully repeated.',
+			);
+		});
+	});
+
+	describe('#getPassphraseFromEnvVariable', () => {
+		const displayName = 'passphrase';
+		beforeEach(() => {
+			delete process.env.PASSPHRASE;
+			return Promise.resolve();
+		});
+
+		after(() => {
+			delete process.env.PASSPHRASE;
```

Comment:
It's best practice to leave your environmental variables as you found them. Check out the pattern used in the 1.0.0 branch: https://github.com/LiskHQ/lisk-commander/blob/1.0.0/test/steps/setup.js#L382-L391